cmd/skipper/secretsFlags.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"strings"
+)
+
+const credentialPathsUsage = "directories or files to watch for credentials to use by bearerinjector filter"
+
+type secretsFlags struct {
+	values []string
+}
+
+func (sf *secretsFlags) String() string {
+	return strings.Join(sf.values, ",")
+}
+
+func (sf *secretsFlags) Set(value string) error {
+	if sf == nil {
+		sf = &secretsFlags{}
```

Comment:
I have seen this in other places, but I don't think that it is useful to initialized this pointer here this way, because once Set exits, the reference on the caller side will still remain nil.